pynio/service.py

```diff
@@ -53,6 +55,26 @@ def stop(self):
         """ Stops the nio Service. """
         self._instance._get('services/{}/stop'.format(self._name))
 
+    def command(self, *args, **kwargs):
+        '''send a command to the service or to the block.
+        To the service:
+            service.command('command')
+        To a block in the service:
+            service.command(block, 'command')
+
+        kwargs are passed onto the request. Some of use are:
+            data: add data onto the request
+            timeout: set the request timeout
+        '''
+        get = self._instance._get
```

Comment:
I've just never seen this before. I would either call the methods directly in line 73. Or if you really want line 73 to be a clean one-liner, then write a method that you pass blk._name, cmd and **kwargs to and then let that method do the rest.